bolt-modules/boltlib/lib/puppet/functions/run_container.rb

```diff
@@ -0,0 +1,161 @@
+# frozen_string_literal: true
+
+require 'bolt/container_result'
+require 'bolt/error'
+require 'bolt/util'
+
+# Run a container.
```

Comment:
The value of the `@return` tag doesn't make it into the docs, so it might be helpful to add that the functions returns the output from the container.